templates/components/form/fields_macros.html.twig

```diff
@@ -702,8 +702,8 @@
       'locked_fields': [],
    }|merge(options) %}
 
-   {% if name in options.locked_fields %}
-      {% set options = options|merge({'locked': true}) %}
+   {% if options.locked_fields[name] is defined %}
+      {% set options = options|merge({'locked': true, 'locked_value': options.locked_fields[name]}) %}
```

Comment:
To prevent any possible BC break, a new option `locked_values` may be added.

```
{% if options.locked_values[name] is defined %}
      {% set options = options|merge({'locked': true, 'locked_value': options.locked_values[name]}) %}
{% elseif name in options.locked_fields %}
      {% set options = options|merge({'locked': true}) %}
{% endif %}
```

Reply:
I really do not see how we can have a bc break here.

Reply:
I still not understand how a plugin would have a bc break on that... Seems a real waste of time.